### DISCUSSION TITLE
Can i edit my data on local server?

### DISCUSSION THREAD

**Discussion by Undore** [thepeacockproject/Peacock#476] [Q&A] [answered]
I found resources folder to be used for this purpose, but files is not readable by text editors. Its not .db or .json, its machine code.
So, can i, for example, unlock all mastery for some map? I already completed it on official servers and i dont want to do it again on PEACOCK server

**Comment by Undore** [comment]
I found that userdata contains JSON files and it works, kinda... But i want to edit everything

**Comment by RDIL** [comment, marked as answer]
Resources contains a mixture of Brotli-compressed json (.br files) and the rest is MessagePack - all your actual data is stored in userdata, and is in JSON format.

**Comment by grappigegovert** [comment]
You might want to check out this:
https://thepeacockproject.org/wiki/features/progression-transfer
